frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/flavor.tsx

```diff
@@ -127,7 +130,7 @@ export const FlavorSelect: React.FC<FlavorProps> = React.memo(
                     {isDefault
                       ? t('kubevirt-plugin~{{flavor}} (default)', { flavor: t(f.toString()) })
```

Comment:
```suggestion
                      ? t('kubevirt-plugin~{{flavor}} (default)', { flavor: flavorDesc} })
```

Reply:
`f.toString()` is the flavor name. `flavorDesc` is the CPU and memory piece, so you can't set `flavor` to that value.